data/campaigns/Secrets_of_the_Ancients/scenarios/21_Against_the_World.cfg

```diff
@@ -635,7 +635,7 @@ We finally made it out of the mountains. We crossed the Ford of Abez late this m
         [/message]
         [message]
             speaker=Ardonna
-            message= _ "That’s why getting through Knalga was so easy."
+            message= _ "That’s why getting through Knalga was so easy. By splitting their forces, they just allowed us to defeat them in detail."
```

Comment:
Sometimes, less is more. I think the original text is better, leaving it to the player to work out how the new information fits into the story.

Reply:
I think the original is fine as well.